std/assembly/util/string.ts

```diff
@@ -599,16 +597,11 @@ function fixmul(a: u64, b: u32): u64 {
   return (high << space | (low & 0xFFFFFFFF) >> revspace) + (low << space >> 31 & 1);
 }
 
+// argument `n` should bounds in [0, 22] range
```

Comment:
nit: "argument `n` must be in range [0, 22]", and perhaps move inside the function